docs/conf.py

```diff
@@ -1,41 +1,74 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
+# conf
+# Yellowbrick documentation build config file, created by sphinx-quickstart
 #
-# yellowbrick documentation build configuration file, created by
-# sphinx-quickstart on Tue Jul  5 19:45:43 2016.
+# Created: Tue Jul 05 19:45:43 2016 -0400
+# Copyright (C) 2016-2019 The scikit-yb developers
+# For license information, see LICENSE.txt
 #
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
+# ID: conf.py [] benjamin@bengfort.com $
+
+"""
+Yellowbrick documentation build config file, created by sphinx-quickstart.
+
+This file is executed with the current directory set to its containing dir
+by ``execfile()``, e.g. the working directory will be yellowbrick/docs.
+Ensure that all specified paths relative to the docs directory are made
+absolute by using ``os.path.abspath``.
+
+Note that not all possible configuration values are present in this
+autogenerated file.
+
+All configuration values have a default; values that are commented out
+serve to show the default.
+
+See: https://www.sphinx-doc.org/en/master/usage/configuration.html
+for more details on configuring the documentation build.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import os
+import sys
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-import os
-import sys
 sys.path.insert(0, os.path.abspath('..'))
 
 # Set the backend of matplotlib to prevent build errors.
 import matplotlib
 matplotlib.use('agg')
 
+# Import yellowbrick information.
 import yellowbrick as yb
 
-# -- General configuration ------------------------------------------------
+##########################################################################
+## General configuration
+##########################################################################
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
+# needs_sphinx = '1.8'
+
+# General information about the project.
+project = 'Yellowbrick'
+copyright = '2016-2019, The scikit-yb developers.'
+author = 'The scikit-yb developers'
```

Comment:
First change in copyright and organization identification.

Reply:
right on!